net/wget/Portfile

```diff
@@ -51,7 +51,6 @@ configure.args          --enable-option-checking \
                         --enable-xattr \
                         --without-ssl \
                         --with-zlib \
-                        --with-metalink \
```

Comment:
Is `--without-metalink` valid? If so, adding that would be preferred. (Combined with `confiure.args-replace` in the variant.)